thefuntasty_danger.gemspec

```diff
@@ -1,13 +1,13 @@
 
 Gem::Specification.new do |spec|
   spec.name        = "thefuntasty_danger"
-  spec.version     = "0.2.0"
+  spec.version     = "0.3.0"
   spec.authors     = ["Matěj Kašpar Jirásek"]
-  spec.email       = ["matej.jirasek@thefuntasty.com"]
+  spec.email       = ["matej.jirasek@futured.app"]
 
   spec.summary     = "The Danger rules we use at The Funtasty"
```

Comment:
Jenom informacne, tenhle `The Funtasty` se nemeni?

Reply:
Kudla pixla. 😂